wordpress-mobile/WordPress-iOS
# Build warnings: fix `xib` warnings

This PR fixes build warnings related to:

* Using `plain` button style (available only on iOS 15+)
* Add correct font to labels with "Automatically Adjust Font" enabled

### To test

1. Do a clean install of WP or JP
1. Login and choose a site
1. ✅ Make sure "Skip" appear correctly
3. Tap "Staying up with Notifications"
1. ✅ Make sure "Not now" appears correctly
1. Create a new draft and tap publish
1. ✅ Make sure the prepublishing nudge (bottom sheet) appears correctly

There's a change on `BloggingPromptsHeaderView.xib` that I'm not sure how to properly test. @wargcm would you mind taking a look, please? I just changed a button style from "Plain" to "Default" (the "Share" button).

## Regression Notes
1. Potential unintended areas of impact
n/a

3. What I did to test those areas of impact (or what existing automated tests I relied on)
n/a

4. What automated tests I added (or what prevented me from doing so)
n/a

PR submission checklist:
